go/fasthttprouter/go.sum

```diff
@@ -0,0 +1,15 @@
+github.com/buaazp/fasthttprouter v0.1.1 h1:4oAnN0C3xZjylvZJdP35cxfclyn4TYkW6Y+DSvS+h8Q=
```

Comment:
keeping the `.sum` is useful, but not for this use-case

Reply:
The whole idea behind the `go.sum` file is to commit it so you have reproducible builds that always use the same dependencies.

Reply:
sure, I know, like a `lock` file in `ruby` ..., but this also means that be versioned ... as this is not an end application, the benefit is not accurate `here`

could it be removed ?

PS : We have no `.sum` files here, I think it's **NOT** needed here :stuck_out_tongue: